examples/gaze-detection/README.md

```diff
@@ -13,12 +13,12 @@ git clone https://github.com/roboflow/inference
 cd inference/examples/gaze-detection
 ```
 
-Next, set up a Python environment and install the required project dependencies:
+Next, set up a Python environment and install the required project dependencies (this assumes running these bash commands from the above terminal window install):
 
 ```bash
 python3 -m venv venv
 source venv/bin/activate
-pip install -r requirements.txt
+pip install -r ../../requirements/_requirements.txt
```

Comment:
Not tested that but I was always thinking that for gaze to work we need also `../../requirements/requirements.gaze.txt` - at least this is suggested by setup.py extras that we created to build libs

Reply:
For now, only these are required in `gaze.py`:
```
import base64
import cv2
import numpy as np
import requests
import os
```
We can assume the user is installing inference, overall, and have them setup the base `_requirements.txt`. In the future, we can decide to scope to individual examples.